### PR TITLE
Remove panics and handle errors gracefully in egress test

### PR DESCRIPTION
**Pull request by fahlmant**
Change errors for a more graceful exit and readable error

### CODE REVIEW

File: pkg/cloudclient/aws/private.go [hunk 1]
```diff
@@ -209,34 +209,39 @@ func (c *Client) validateEgress(ctx context.Context, VPCSubnetID string) error {
...
 
 	fmt.Println("Terminating instance")
 	if err := terminateEC2Instance(c.ec2Client, instanceID); err != nil {
-		panic(err)
+		fmt.Printf("Error terminating instances: %s\n", err.Error())
+		return err
 	}
 	if len(unreachableEndpoints) > 0 {
-		return errors.New(fmt.Sprintf("multiple target unreachable %q", unreachableEndpoints))
+		return fmt.Errorf("multiple target unreachable %q", unreachableEndpoints)
```
Comment (tiwillia) [inline pkg/cloudclient/aws/private.go:244]: nitpick: s/target/targets/

Another nitpick: Inconsistent capitalization of errors messages throughout.

Not worth blocking the MR.

Comment (fahlmant) [reply pkg/cloudclient/aws/private.go:244]: Errorf gives a warning that says not to capitalize, while Printf doesn't :(

**Pull request merged**